boskos/cmd/janitor/cleaners.go

```diff
@@ -0,0 +1,11 @@
+package cleaners
+
+import (
+	"google.golang.org/api/compute/v1"
+)
+
+type cleaners interface {
```

Comment:
nit: this shoulld probably be singular, and probably also wants to be public (i.e. capitalized)